pydalboard/modules/delay.py

```diff
@@ -38,8 +37,10 @@ def process(self, input: np.ndarray, signal_info: SignalInfo) -> np.ndarray:
         output = input
 
         if len(self.memory) > 0:
-            delayed_sample = self.memory[0] # Oldest sample in the deque
-            output += delayed_sample * self.params.feedback # Mix the input with the delayed one
+            delayed_sample = self.memory[0]  # Oldest sample in the deque
+            output += (
```

Comment:
Are you using a tuple here? Why are you using ()? I don't understand

Reply:
That was a choice by my linter. With the comment at the end, the line was considered too long, and was split into 3. In python, you can do this by opening parenthesis, otherwise, `output +=` would be considered a syntax error.

In python, an expression in parenthesis does not become a tuple, you have to add an extra `,` like so:

```python
not_a_tuple = (value)
a_tuple = (value,)
```.

I can move the comment above the line, or reword it shorter.